QUEST.tsv

```diff
@@ -1016,13 +1016,13 @@ QUEST_20150317_001015	Damn, I thought it would die after getting its throat cut.
 QUEST_20150317_001016	I was late this time. Well, it doesn't matter that much. {nl}Soon my masterpiece will be complete. I hope you guys like it.
 QUEST_20150317_001017	Unknown Demon
 QUEST_20150317_001018	Ah… I almost forgot. I'll be taking this guy along!
-QUEST_20150317_001019	I can feel the power of Laima
-QUEST_20150317_001020	The pain I haven't felt for a long time…
+QUEST_20150317_001019	I can feel the power of Laima.
+QUEST_20150317_001020	The pain I haven't felt for a long time...
 QUEST_20150317_001021	Even so, I will not waste my power on you.
-QUEST_20150317_001022	I guess the real revelation is in the cathedral? {nl}That Laima could only prepare this much. Such a pity.
-QUEST_20150317_001023	This is the crest of the space…
+QUEST_20150317_001022	I guess the real revelation is in the chapel? {nl}That Laima could only prepare this much. Such a pity.
+QUEST_20150317_001023	This is the crest of space...
 QUEST_20150317_001024	We will need the Revelator.
-QUEST_20150317_001025	The Revelator came here alone. {nl}Just what I wanted.
+QUEST_20150317_001025	The Revelator came here alone. {nl}Just what I exactly wanted.
```

Comment:
"Just what I exactly wanted." is redundant, and doesn't sound right. "Exactly what I wanted/Just what I wanted." are better choices.